samples/cpp/tutorial_code/gapi/face_beautification/face_beautification_old.cpp

```diff
@@ -0,0 +1,771 @@
+// This file is part of OpenCV project.
```

Comment:
Can you please define how this file is used? I mean the problem is the name it has: (`*_old`). Is it actual file?

Reply:
Yeah: this file is for giving an example of non-Streaming graph processing. In other words - the example how we had to implement the videostream processing without Streaming-API

Reply:
Code was reorganized, thank you